Assets/stylesheet.css

```diff
@@ -0,0 +1,264 @@
+* {
```

Comment:
This is valid CSS3 grammar, but with custom attributes and our scalars lack unit specifications, because we only support pixels (at least for now).